src/main/java/frc/robot/subsystems/NAVX.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot.subsystems;
+
+//why is this not importing?
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.I2C.Port;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotContainer;
+import java.nio.ByteBuffer;
+import edu.wpi.first.hal.I2CJNI;
+import edu.wpi.first.wpilibj.I2C.Port;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DigitalSource;
+import edu.wpi.first.wpilibj.I2C;
+
+@SuppressWarnings("unused")
+public class NAVX extends SubsystemBase {
+
+    private AHRS navx;
+    private AnalogInput irSensor1;
+    private LiDAR lidar;
+
+    private RobotContainer container; // This classes reference to the RobotContainer
+
+    public NAVX(RobotContainer container) {
+        this.container = container;
```

Comment:
You don't use container anywhere - remove argument for RobotContainer as well as private variable.